### PR TITLE
Add snow in upgrade doc

### PR DESCRIPTION
**Pull request by jiayiwang7**
*Issue #, if available:*

https://github.com/aws/eks-anywhere/issues/1111

*Description of changes:*

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/tasks/cluster/cluster-upgrades/vsphere-and-cloudstack-upgrades.md [hunk 1]
```diff
@@ -47,6 +47,8 @@ version.
 Irrespective of a Kubernetes version change, the upgrade command will always upgrade the internal EKS
 Anywhere components mentioned above to their latest available versions. All upgrade changes are backwards compatible.
 
+Specifically for Snow provider, see [Upgrade EKS Anywhere AMIs in Snowball Edge devices](https://docs.aws.amazon.com/snowball/latest/ug/whatissnowball.html) to upgrade and use a new Admin instance which contains a newer version of EKS Anywhere CLI binary.
```
Comment (chrisnegus) [inline docs/content/en/docs/tasks/cluster/cluster-upgrades/vsphere-and-cloudstack-upgrades.md:50]: Could you be more specific about the new Snow Admin instances only being needed when upgrading to new versions of EKS Anywhere and that upgrades of other components can be done as described in this document?

**Review by chrisnegus — COMMENTED** [review]
Looks good. Just one comment.

File: docs/content/en/docs/tasks/cluster/cluster-upgrades/vsphere-and-cloudstack-upgrades.md (hunk 1 above)
Comment (jiayiwang7) [reply docs/content/en/docs/tasks/cluster/cluster-upgrades/vsphere-and-cloudstack-upgrades.md]: done

**Review by chrisnegus — COMMENTED** [review]
/lgtm

**Pull request merged**